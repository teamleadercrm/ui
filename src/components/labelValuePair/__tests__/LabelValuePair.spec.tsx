import React from 'react';
import { render } from '@testing-library/react';
import { LabelValuePair, LabelValuePairGroup } from '..';

describe('Component - LabelValuePair', () => {
  it('renders', async () => {
    const { asFragment } = render(
      <LabelValuePairGroup>
        <LabelValuePair>
          <LabelValuePair.Label>Label</LabelValuePair.Label>
          <LabelValuePair.Value>Value</LabelValuePair.Value>
        </LabelValuePair>
      </LabelValuePairGroup>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
