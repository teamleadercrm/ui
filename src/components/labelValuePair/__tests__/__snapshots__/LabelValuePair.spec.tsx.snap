// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - LabelValuePair renders 1`] = `
<DocumentFragment>
  <div
    class="box"
    data-teamleader-ui="label-value-pair-group"
  >
    <h4
      class="box margin-bottom-2 heading heading-4 darkest neutral"
      data-teamleader-ui="heading"
    />
    <div
      class="box display-flex flex-direction-row margin-bottom-1"
      data-teamleader-ui="label-value-pair"
    >
      <h5
        class="box margin-right-2 padding-bottom-1 padding-top-1 heading heading-5 darkest teal overflow-multiline"
        data-teamleader-ui="heading"
        style="flex: 0 0 40%;"
      >
        Label
      </h5>
      <div
        class="box display-flex justify-content-flex-start padding-bottom-1 padding-top-1 text-align-left"
        style="flex: 1; overflow: hidden;"
      >
        Value
      </div>
    </div>
  </div>
</DocumentFragment>
`;
